cinema/models.py

```diff
@@ -30,28 +36,45 @@ class Actor(models.Model):
     def __str__(self):
         return self.first_name + " " + self.last_name
```

Comment:
Use property  `full_name` here, please

Reply:
you mean to delete str method and just use lines 39-41?